cmd/gf/internal/cmd/cmd_gen_service.go

```diff
@@ -354,3 +403,30 @@ func (c cGenService) replaceGeneratedServiceContentGFV2(in cGenServiceInput) (er
 		return content
 	}, in.DstFolder, "*.go", false)
 }
+
+// getDstFileNameCase call gstr.Case* function to convert the s to specified case.
+func getDstFileNameCase(str, caseStr string) string {
```

Comment:
Please put this function as a method of `cGenService`.

Reply:
Fixed, thanks for the reminder